### PR TITLE
Add type to make getPath call types

### PR DESCRIPTION
**Pull request by moshfeu**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -57,6 +57,7 @@ export function getSaveGamesFolder(): string {
  * Returns path for type or undefined if name is invalid
  * @param name folder type (see README)
  */
+type Folder = 'home' | 'appData' | 'appdata' | 'userData' | 'desktop' | 'documents' | 'downloads' | 'music' | 'pictures' | 'videos' | 'cache' | 'savegames';
```
Comment (kingjan1999) [inline src/index.ts]: Can you insert one or more line break(s) here? The line is quite long.

File: src/index.ts [hunk 2]
```diff
@@ -57,6 +57,7 @@ export function getSaveGamesFolder(): string {
...
 export default function getPath(name: string): string|undefined {
```
Comment (kingjan1999) [inline src/index.ts]: I think you have to replace `name: string` with `name: Folder` to use the type declared above

**Review by kingjan1999 — CHANGES_REQUESTED** [review]
Thank you very much! Just two minor comments

**Comment by moshfeu** [comment]
Done. Thanks :)

**Review by kingjan1999 — APPROVED** [review]

**Comment by kingjan1999** [comment]
Thank you, merged!

**Pull request merged**